src/components/Bookmarks.vue

```diff
@@ -176,6 +177,52 @@ export default {
                 title: '',
                 url: '',
                 icon: '',
+                iconType: 'font'
+            },
+            iconMapping: {
+                // Social Media
+                'facebook.com': 'fab fa-facebook',
+                'twitter.com': 'fab fa-twitter',
+                'x.com': 'fab fa-x-twitter',
+                'instagram.com': 'fab fa-instagram',
+                'linkedin.com': 'fab fa-linkedin',
+                'youtube.com': 'fab fa-youtube',
+                'tiktok.com': 'fab fa-tiktok',
```

Comment:
should we discourage usage of tiktok.com and facebook by not supporting their icon by default?